src/utils/getTreasuryBalances.mjs

```diff
@@ -0,0 +1,128 @@
+import env from 'dotenv';
+import alasql from 'alasql';
```

Comment:
OK to use alasql as interim solution now.
Recommendation: use GraphQL

https://thegraph.com/en/

Dubbed as "the Google for Blockchain", GraphQL is indexing the events emitted from Ethereum contracts and a few other chains. Interestingly their website shows an example of indexing a DAO contract events. Underneath it is just a PostgreSQL DB wrapped with indexers and queries.